pandas/tests/arrays/test_ndarray_backed.py

```diff
@@ -0,0 +1,89 @@
+"""
+Tests for subclasses of NDArrayBackedExtensionArray
+"""
+import numpy as np
+import pytest
+
+from pandas import (
+    CategoricalIndex,
+    date_range,
+)
+from pandas.core.arrays import (
+    Categorical,
+    DatetimeArray,
+    PandasArray,
+    PeriodArray,
+    TimedeltaArray,
+)
+
+
+@pytest.fixture(
+    params=[Categorical, DatetimeArray, TimedeltaArray, PeriodArray, PandasArray]
```

Comment:
could move to conftest?

Reply:
woops, actually should have removed this since it isnt used